test/test_cache.py

```diff
@@ -167,57 +166,65 @@ def test_pm_only(self):
 
 
 class TestMethods(unittest.TestCase):
-
     @classmethod
     def setUpClass(cls):
-        cls.gc = Geocaching()
-        cls.gc.login(_username, _password)
+        cls.gc = Geocaching(_session=session)
         cls.c = Cache(cls.gc, "GC1PAR2")
-        cls.c.load()
+        with recorder.use_cassette('cache_setup'):
```

Comment:
What do you think about using single setup (login) cassette for all logins?

Reply:
I think I know what you mean, but can you clarify so there is no confusion?

Reply:
There are multiple cassettes which starts by user login procedure and does not differ in any way, or only slightly. It may be nice to record only one cassette containing only `gc.login(_username, _password)` call and then reuse it in all these tests.